introduction.md

```diff
@@ -1 +1,4 @@
-Hello, my name is David Aquino.  I am a Software Engineer at AOL. My favorite Code School path is Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?